PythonApp2App/AppClient/AppClient.py

```diff
@@ -0,0 +1,73 @@
+from WinRT.ApplicationModel import AppService
```

Comment:
This is my unit test code.  This isn't really suitable for a sample.  I think you should simplify it bit so the user can have an easier time understanding.